docs/tracking-javascript-guide.md

```diff
@@ -693,6 +693,52 @@ _paq.push(['trackPageView']);
 
 ## Multiple Piwik trackers
 
+### If you use Piwik 2.16.2 or later
+
+It is possible to track your analytics data into multiple separate Piwik trackers. You may record your data either in the same Piwik server (into a different website ID) or you may record a duplicate of your data into another Piwik server altogether. 
+
+The following example shows how to use `addTracker` to store all of your data in another Piwik server hosted at `analytics.example.com`:
+
+```html
+<script type="text/javascript">
+  var _paq = _paq || [];
+  _paq.push(['trackPageView']);
+  _paq.push(['enableLinkTracking']);
+  
+  (function() {
+    var u="//{$PIWIK_URL}/";
+    _paq.push(['setTrackerUrl', u+'piwik.php']);
+    _paq.push(['setSiteId', {$IDSITE}]);
+    
+    // Also send all of the tracking data to this other Piwik server
+    // By default, the data will be stored into the same website ID as the main tracker (see `setSiteId` call above)
+    _paq.push(['addTracker', 'https://analytics.example.com']);
```

Comment:
I'm not sure if we need to mention the case without siteId. I think it would be best practice and less confusing by only mentioning the example with the same siteId. It will make it also more clear and lead to less errors if eg someone changes the siteId of the initial tracker etc